cms/grading/tasktypes/Batch.py

```diff
@@ -262,6 +265,7 @@ def evaluate(self, job, file_cacher):
             commands,
             job.time_limit,
             job.memory_limit,
+            max_processes,
```

Comment:
yeah, feel free to use whichever idiom you prefer. fun fact: __dict__.get() in cpython is slower than the if/else. I'm not particularly attached to either.

Hmm, I only just noticed that the job-specific args weren't kwargs (...wtf?) so yeah, okay, fair point; don't make this a kwarg. Seems legit.